.gitignore

```diff
@@ -8,12 +8,12 @@
 /_yardoc/
 /bundler.d/
 /config/database.yml
+/spec/dummy/db/schema.rb
```

Comment:
I'd prefer to leave this where it was in alphabetical order

Reply:
👍